.rubocop.yml

```diff
@@ -0,0 +1,14 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2017-06-02 23:12:20 -0700 using RuboCop version 0.45.0.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 23
+# Configuration parameters: AllowHeredoc, AllowURI, URISchemes, IgnoreCopDirectives.
+# URISchemes: http, https
+Metrics/LineLength:
+  Max: 120
```

Comment:
i really like 80, but we can increase a little to 90 i guess. The problem is if it's too long you can't use split views in any ide also affects github splitview of commits